iothub/service/samples/getting started/ReadD2cMessages/ReadD2cMessages.csproj

```diff
@@ -11,4 +11,11 @@
     <PackageReference Include="CommandLineParser" Version="2.9.1" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(Configuration)' == 'Release'">
+    <PackageReference Include="Microsoft.Azure.Devices" Version="1.38.1" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(Configuration)' == 'Debug'">
+    <ProjectReference Include="..\..\..\src\Microsoft.Azure.Devices.csproj" />
```

Comment:
This sample doesn't use our service client IIRC. You should be able to skip adding this dependency because of that.